scripts/quests/ahtUrhgan/Waking_the_Colossus.lua

```diff
@@ -0,0 +1,392 @@
+-----------------------------------
+-- Log ID: 6, Quest ID: 74
+-- Naja Salaheem, !pos 22.700 -8.804 -45.591 50
+-- Imperial Whitegate, !pos 152.8295 -2.2 0.0613 50
+-- Audience Chamber, !pos 0 -5 66 243
+-- Halver !pos 2.4 0 0 233
+-- Iron Eater !pos 91 -19 1 237
+-- Kupipi !pos 2 0 30 242
+-- Audience Chamber, !pos 0 -5 66 243
+-- Imperial Whitegate, !pos 152.8295 -2.2 0.0613 50
+-- Acid-Eaten Door, !pos 271.9045 -32 -88 61
+-- Blank Lamp, !pos 209.3953 0.05 20.0207 72
+-- Runic Seal, !pos 125.4569 0 19.9914 72
+-- Imperial Whitegate, !pos 152.8295 -2.2 0.0613 50
+-- Naja Salaheem, !pos 22.700 -8.804 -45.591 50
+-----------------------------------
+local ahtUrhganID = zones[xi.zone.AHT_URHGAN_WHITEGATE]
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.AHT_URHGAN, xi.quest.id.ahtUrhgan.WAKING_THE_COLOSSUS)
+
+quest.reward =
+{
+    title = xi.title.HEIR_OF_THE_BLESSED_RADIANCE,
+    item  = { xi.item.IMPERIAL_GOLD_PIECE }
+}
+
+local rewardItems =
+{
+    [0] = xi.item.COLOSSUSS_MANTLE,
+    [1] = xi.item.COLOSSUSS_EARRING,
+    [2] = xi.item.COLOSSUSS_TORQUE,
+}
+
+local function getRewardMask(player)
+    local rewardMask = 0
+
+    for bitNum, itemId in pairs(rewardItems) do
+        if player:hasItem(itemId) then
+            rewardMask = utils.mask.setBit(rewardMask, bitNum, true)
+        end
+    end
+
+    if player:hasSpell(xi.magic.spell.ALEXANDER) then
+        rewardMask = utils.mask.setBit(rewardMask, 4, true)
+    end
+
+    return rewardMask
+end
+
+local function giveQuestReward(player, eventOption)
+    local wasRewarded = true
+
+    if eventOption <= 3 then
+        wasRewarded = npcUtil.giveItem(player, rewardItems[eventOption - 1])
+    elseif eventOption == 4 then
+        npcUtil.giveCurrency(player, 'gil', 10000)
+    elseif eventOption == 5 then
+        player:addSpell(xi.magic.spell.ALEXANDER)
+        player:messageSpecial(ahtUrhganID.text.ALEXANDER_UNLOCKED, 0, 0, 1)
+    end
+
+    return wasRewarded
+end
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+                player:getCurrentMission(xi.mission.log_id.TOAU) == xi.mission.id.toau.ETERNAL_MERCENARY
+        end,
+
+        [xi.zone.AHT_URHGAN_WHITEGATE] =
+        {
+            ['Imperial_Whitegate'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        quest:getVar(player, 'Prog') == 1 and
+                        not player:hasKeyItem(xi.ki.IMPERIAL_MISSIVE)
+                    then
+                        return quest:progressEvent(926, { text_table = 0 })
+                    end
+                end,
+            },
+
+            onTriggerAreaEnter =
+            {
+                [13] = function(player, triggerArea)
+                    if quest:getVar(player, 'Prog') == 0 then
+                        return quest:progressEvent(925, { text_table = 0 })
```

Comment:
`return quest:progressEvent(925)`

Reply:
you use [ text_table = 0 } if it uses the other dat in the zone, 
similates the 8th param as 0 to activate